config/environments/development.rb

```diff
@@ -1,28 +1,41 @@
+require "active_support/core_ext/integer/time"
+
 Rails.application.configure do
   config.cache_classes = false
   config.eager_load = false
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local = true
+
+  # Tell Rails not to send server timing headers
+  config.server_timing = false
```

Comment:
Server Timing headers provide a header with how long the view, SQL, etc each took.

From the [7.0.0 release notes](https://github.com/rails/rails/releases/tag/v7.0.0):

>Add Server-Timing middleware
>Server-Timing specification defines how the server can communicate to browsers performance metrics
about the request it is responding to.
> The ServerTiming middleware is enabled by default on development environment by default using the
config.server_timing setting and set the relevant duration metrics in the Server-Timing header
> The full specification for Server-Timing header can be found in: https://www.w3.org/TR/server-timing/#dfn-server-timing-header-field

I disabled it because:

1) We definitely don't need these kinds of metrics, even in development
2) It tells Rails to do more work, so why not tell it to do less?